app/validation/integer.py

```diff
@@ -0,0 +1,22 @@
+from app.validation.i_validator import IValidator
+from app.validation.validation_result import ValidationResult
+
+
+class Integer(IValidator):
```

Comment:
I really think this should be called IntegerValidator or IntegerValidationRule or something, as Integer clashes with the Type Integer.